docs/expo.md

```diff
@@ -0,0 +1,163 @@
+  const UIApplicationState appState = application.applicationState;
+  [AEPMobileCore registerExtensions:@[
+      AEPMobileLifecycle.class, AEPMobileIdentity.class,
+      AEPMobileEdgeIdentity.class, AEPMobileEdge.class,
+      AEPMobileEdgeConsent.class, AEPMobileEdgeBridge.class
+    ]
+  completion:^{
+    if (appState != UIApplicationStateBackground) {
+      [AEPMobileCore lifecycleStart:nil];
+      }
+      }];
+
+  return [super application:application didFinishLaunchingWithOptions:launchOptions];
+}
+
+@end
+
+```
+
+> To enable the Lifecycle metrics, [implement the Lifecycle APIs](./packages/core/README.md#lifecycle)
+
+> Hint : While running iOS application after Adobe Experience Platform SDK installation. If you have build error that states:
+> "ld: warning: Could not find or use auto-linked library 'swiftCoreFoundation'"
+> This is because Adobe Experience Platform SDK now requires the app uses swift interfaces. Add a dummy .swift file to your project to embed the swift standard libs. See the SampleApp presented in this repo for example.
+
+##### **Android:** _(Kotlin)_
+
+```kotlin
+// MainApplication.kt
+import com.adobe.marketing.mobile.AdobeCallback;
+import com.adobe.marketing.mobile.Extension;
+import com.adobe.marketing.mobile.LoggingMode;
+import com.adobe.marketing.mobile.MobileCore;
+import com.adobe.marketing.mobile.Lifecycle;
+import com.adobe.marketing.mobile.Signal;
+import com.adobe.marketing.mobile.Edge;
+import com.adobe.marketing.mobile.edge.consent.Consent;
+```
+
+```kotlin
+class MainApplication : Application(), ReactApplication {
+  ...
+  override fun onCreate() {
+    super.onCreate()
+    ...
+
+    MobileCore.setApplication(this)
+    MobileCore.setLogLevel(LoggingMode.VERBOSE)
+    MobileCore.configureWithAppID("YOUR-APP-ID")
+    val extensions: List<Class<out Extension?>> = Arrays.asList(
+      Lifecycle.EXTENSION,
+      Signal.EXTENSION,
+      Edge.EXTENSION,
+      Consent.EXTENSION,
+      com.adobe.marketing.mobile.Identity.EXTENSION
+    )
+    MobileCore.registerExtensions(extensions,
+      AdobeCallback { o: Any? ->
+        MobileCore.lifecycleStart(
+          null
+        )
+      })
+  }
+}
+```
+
+> To enable the Lifecycle metrics, [implement the Lifecycle APIs](./packages/core/README.md#lifecycle)
+
+# Guide for Bare React Native apps using Expo modules
+
+## Overview
+Bare React Native workflows can be integrated with Expo SDKs by using the `install-expo-modules` command. This allows you to use Expo modules in your app.
+- To use Expo modules in your app, you will need to install and configure the expo package.
+- The expo package has a small footprint; it includes only a minimal set of packages that are needed in nearly every app and the module and autolinking infrastructure that other Expo SDK packages are built with.
+- Once the expo package is installed and configured in your project, you can use `npx expo install` to add any other Expo module from the SDK.
```

Comment:
To use Expo modules in an app, installation and configuration of the `expo` package is required. The `expo` package has a small footprint; it includes only a minimal set of packages needed in nearly every app and the module and autolinking infrastructure that other Expo SDK packages are built with. After the `expo` package is installed and configured in the project, `npx expo install` can be used to add any other Expo module from the SDK.